packages/web/components/complex/pools-table.tsx

```diff
@@ -176,10 +178,14 @@ export const PoolsTable = (props: PropsWithChildren<PoolsTableProps>) => {
     }
   );
 
-  const poolsData = useMemo(
-    () => poolsPagesData?.pages.flatMap((page) => page?.items) ?? [],
-    [poolsPagesData]
-  );
+  const poolsData = useMemo(() => {
+    const allItems =
+      poolsPagesData?.pages.flatMap((page) => {
+        return page?.items.filter((item) => !FILTERABLE_IDS.includes(item.id)); // Filter out ids in FILTERABLE_IDS
+      }) ?? [];
+
+    return allItems;
+  }, [poolsPagesData]);
```

Comment:
Do we only want to filter this pool in the pools table? If not, I suggest modifying https://github.com/osmosis-labs/osmosis-frontend/blob/288c023d8100418b8f5a2b607f532dd6e0014112/packages/server/src/queries/complex/pools/index.ts#L89 instead, so the changes propagate to all components that rely on pools.